source/help/messaging/sending-messages.md

```diff
@@ -22,4 +22,4 @@ Edit a message by clicking the **[...]** icon next to any message that you’ve
 Delete a message by clicking the **[...]** icon next to any message that you've sent, then click **Delete**. System and Team Admins can delete any message on their system or team.
 
 ## Linking to a message
-The **Permalink** feature creates a link to any message. Get the permalink to any message by clicking the **[...]** icon next to the a message  > **Permalink** > **Copy Link**. Sharing this link will let other users view the linked message in the Message Archives. If the link is to a message in a Public Channel, any user of the team can see the message. If the link is to a message in a Private Group, only other users in the group will be able to see the message. 
+The **Permalink** feature creates a link to any message. Get the permalink to any message by clicking the **[...]** icon next to the a message  > **Permalink** > **Copy Link**. Sharing this link will let other users view the linked message in the Message Archives. If the link is to a message in a Public Channel, any user of the team can see the message. If the link is to a message in a Private Channel, only other users in the private channel can see the message. 
```

Comment:
"If the link is to a message in a Private Channel, only other users in the private channel can see the message. " --> Shouldn't the second "private channel" be capitalized like the first?